_im-plugin/index-rollups/index.md

```diff
@@ -454,3 +454,265 @@ POST example_rollup/_search
   }
 }
 ```
+
+## The doc_count field
+
+The `doc_count` field in bucket aggregations holds the number of documents collected in each bucket. When calculating the bucket's `doc_count`, the number of documents is incremented by the number of the pre-aggregated documents in each summary document. The `doc_count` returned from rollup searches represents the total number of matching documents from the source index. Thus, the document count for each bucket is the same whether you search the source index or the rollup target index.
```

Comment:
Instead of "holds", do we mean "displays" or "shows"?

Reply:
No, "displays" is not the right word. I can change to "contains".